src/utils/prefs.js

```diff
@@ -40,4 +41,8 @@ const prefs = new PrefsHelper("devtools", {
   fileSearchRegexMatch: ["Bool", "debugger.file-search-regex-match"]
 });
 
-module.exports = { prefs };
+function setDebuggerVersion() {
+  prefs.debuggerVersion = DEBUGGER_VERSION;
+}
+
+module.exports = { prefs, setDebuggerVersion };
```

Comment:
I think the best place to implement the clear function would be in `devtool-module` -- i added a todo note, shall i make a separate issue?